composer.json

```diff
@@ -39,7 +39,7 @@
     "require": {
         "php": ">=5.6",
         "ext-openssl": "*",
-        "lcobucci/jwt": "^3.2",
+        "lcobucci/jwt": "^3.2|^4.0",
```

Comment:
This change is not compatible with version 3.3.3:
Notice: Object of class DateTimeImmutable could not be converted to int in .../vendor/lcobucci/jwt/src/Builder.php line 188
You should update this to "^3.4|^4.0"

Reply:
Right, the version detection logic will be fixed in #805. Thanks for the report @JochenJung